BetterLabelTests/UIButtonExtensionTests.swift

```diff
@@ -0,0 +1,20 @@
+//
+//  UIButtonExtensionTests.swift
+//  BetterLabelTests
+//
+//  Created by Lukáš Hromadník on 18/06/2019.
+//  Copyright © 2019 Jakub Olejník. All rights reserved.
+//
+
+import XCTest
+@testable import BetterLabel
+
+final class UIButtonExtensionTests: XCTestCase {
+    func testCallability() {
+        let label = BetterLabel()
+        let button = UIButton()
+        button.setBetterLabel(label, for: .normal)
+    }
+    
+    
```

Comment:
I think that those spaces should be deleted.

Reply:
Yes, you're right. I've removed it